### ISSUE TITLE
unicode problem

### ISSUE THREAD

**Issue by regisz** [compilelife/ulauncher-searchfile#1]
Hi,

I try to search anything with Hungarian (unicode) characters (á, é, etc.), then it does not work. But, If I try to do it in a terminal, then ok.

**Comment by compilelife** [comment]
Sorry about my poor knowledge on Hungarian characters and can't offer any help currently.

It looks like all UTF8 characters work fine on my computer, even with Chinese words.

This project mainly shares a workaround to search file in ulauncher. Hopes the author would consider supporting searching feature in more effective way.

**Comment by regisz** [comment]
Of course, if Chinese characters work then the Hungarian ones should work too :). (In Hungarian language there are only 9 special characters next to English ones :) ).

Maybe, is this a ULauncher problem? (Because native locate handles the Hungarian characters.)
